crates/client/src/lib.rs

```diff
@@ -71,12 +72,12 @@ impl<P: PackageStorage, C: ContentStorage> Client<P, C> {
     pub async fn publish(&self, signing_key: &signing::PrivateKey) -> ClientResult<()> {
         let info = self
             .packages
-            .load_publish()
+            .load_publish("dogfood")
```

Comment:
The `RegistryStorage` is already per-registry, so no need for a hardcoded `"dogfood"` here and below (see my comments on the `RegistryStorage` trait).